app/src/components/CommitTeaser.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import UserCard from "../components/UserCard.svelte";
```

Comment:
The imported component lives in the same directory. Do we need the exta `../componentes` prefix?